app/jobs/visitor_cleaner_job.rb

```diff
@@ -0,0 +1,9 @@
+class VisitorCleanerJob < ApplicationJob
+  queue_as :default
+
+  def perform(*args)
+    days_to_remove = 5.days.ago
```

Comment:
Hi. I would extract to a constant

```ruby
class VisitorCleanerJob < ApplicationJob
  NB_DAYS = 5 # or even 5.days.ago
  
  def perform(*args)
    days_to_remove = NB_DAYS.days.ago
  end
end
```

Reply:
this could also be an ENV configuration variable.